routers/repo/issue_dependency_remove.go

```diff
@@ -0,0 +1,61 @@
+// Copyright 2017 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package repo
+
+import (
+	"fmt"
+	"net/http"
+	"strconv"
+
+	"code.gitea.io/gitea/models"
+	"code.gitea.io/gitea/modules/context"
+)
+
+// RemoveDependency removes the dependency
+func RemoveDependency(c *context.Context) {
+	depID, err := strconv.ParseInt(c.Req.PostForm.Get("removeDependencyID"), 10, 64)
+	if err != nil {
```

Comment:
Please minimize that as shown above

Reply:
Don't think it is worth it, I would need to declare both `depID` and `err` before calling the function which wouldn't result in better readability.